components/LoginForm/LoginForm.js

```diff
@@ -30,16 +31,20 @@ LoginForm.propTypes = {
     email: string,
     password: string,
   }),
+  redirectFunction: func,
 };
 
 LoginForm.defaultProps = {
   initialValues: {
     email: '',
     password: '',
   },
+  redirectFunction: noop,
 };
 
-function LoginForm({ initialValues, login, onSuccess }) {
+function LoginForm({ initialValues, login, onSuccess, redirectFunction }) {
+  const isInLoginModal = redirectFunction.name !== 'noop';
```

Comment:
Hmmm this doesn't seem right. This places business logic into a reusable component. To keep in the spirit of reusability, I think we'll want to pass `theme` prop which conditionally adjusts the button's theme (and will work for any other dynamically styled bits)

Reply:
yep I agree. will be addressing this.